pkg/visor/visorconfig/config.go

```diff
@@ -33,12 +33,10 @@ func MakeBaseConfig(common *Common) *V1 {
 		RouteFinderTimeout: DefaultTimeout,
 	}
 	conf.Launcher = &V1Launcher{
-		Discovery: &V1ServiceDisc{
-			ServiceDisc: skyenv.DefaultServiceDiscAddr,
-		},
-		Apps:       nil,
-		ServerAddr: skyenv.DefaultAppSrvAddr,
-		BinPath:    skyenv.DefaultAppBinPath,
+		ServiceDisc: skyenv.DefaultServiceDiscAddr,
```

Comment:
@ersonp we do need to define a migration somewhere for that, right?

Reply:
I think we just need to update the already existing migration in `parse.go/updateUrls()`

Reply:
Just tested by running a visor from this PR on config from `master`, and we do not need any extra migration. The already existing one takes care of it.